config/config_test.go

```diff
@@ -77,3 +80,23 @@ func TestIsEnabledABSearch(t *testing.T) {
 		})
 	})
 }
+
+func TestValidatePrivatePrefix(t *testing.T) {
+	Convey("given an empty private path prefix", t, func() {
+		Convey("validatePrivatePrefix returns the empty private path prefix", func() {
+			So(validatePrivatePrefix("a-prefix"), ShouldEqual, "/a-prefix")
```

Comment:
I think this is meant to be the empty string?
```suggestion
			So(validatePrivatePrefix(""), ShouldEqual, "")
```